doc/classes/NavigationAgent3D.xml

```diff
@@ -36,7 +37,7 @@
 		<method name="get_next_location">
 			<return type="Vector3" />
 			<description>
-				Returns a [Vector3] in global coordinates, that can be moved to, making sure that there are no static objects in the way. If the agent does not have a navigation path, it will return the origin of the agent's parent.
+				Returns a [Vector3] in global coordinates, that can be moved to, making sure that there are no static objects in the way. If the agent does not have a navigation path, it will return the origin of the agent's parent. The use of this function once every physics frame is required to update the internal path logic of the NavigationAgent.
```

Comment:
```suggestion
				Return a [Vector3] in global coordinates that can be moved to, making sure that there are no static objects in the way. If the agent does not have a navigation path, it will return the origin of the agent's parent. The use of this function once every physics frame is required to update the internal path logic of the NavigationAgent.
```